circle.yml

```diff
@@ -0,0 +1,24 @@
+machine:
```

Comment:
I know, I set this up for another repo and was very :( Come on Circle, get your shit together :P